OfficeApp3Web/Content/AppV2.css

```diff
@@ -545,6 +545,18 @@ li.listItem:hover{
     font-weight:600;
 }
 
+#sampleSelector:focus {
+    outline-color: black;
+    outline-style: dotted;
+    outline-width: 1.5px;
+}
+
+#groupSelector:focus {
+    outline-color: black;
+    outline-style: dotted;
+    outline-width: 1.5px;
+}
+
 @media screen and (-ms-high-contrast:none){

```

Comment:
need to update border:1px solid #9E9E9E; to use outline as well in line 574 and 579